### PR TITLE
Service event extension

### PR DESCRIPTION
**Pull request by JanneAHeikkinen**
ServiceEvent extension (incl. StructureDefinition intro template) for FiSchedulingAppointment.

### CODE REVIEW

File: input/pagecontent/StructureDefinition-fi-scheduling-service-event-intro.md [hunk 1]
```diff
@@ -0,0 +1 @@
+### Scope and Usage
```
Comment (mrinnetmaki) [inline input/pagecontent/StructureDefinition-fi-scheduling-service-event-intro.md:1]: Are you planning to add any content here? I believe it should
- refer to the base profiles (https://hl7.fi/fhir/finnish-base-profiles/StructureDefinition-fi-base-encounter.html#relation-to-finnish-palvelutapahtuma)
- state that "when the service event id (*palvelutapahtuman tunniste*) is included into the Appointment resource, it SHALL be done with this extension
- potentially explain that there are other mechanisms for communicating this information too, like including it into the Provenance, as Kanta does, or referring to the Appointment resource from the Encounter.

**Review by mrinnetmaki — CHANGES_REQUESTED** [review]
Suomi mieluiten kursiivilla

File: input/fsh/extensions/ServiceEvent.fsh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Extension: ServiceEvent
+Id: fi-scheduling-service-event
+Description: "Extension for service event id (palvelutapahtuman tunniste)."
```
[the remark this answers is not in the document]
Comment (JanneAHeikkinen) [reply input/fsh/extensions/ServiceEvent.fsh]: Fixed

File: input/pagecontent/StructureDefinition-fi-scheduling-service-event-intro.md (hunk 1 above)
Comment (JanneAHeikkinen) [reply input/pagecontent/StructureDefinition-fi-scheduling-service-event-intro.md]: Added some content.

File: input/fsh/extensions/ServiceEvent.fsh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Extension: ServiceEvent
 Id: fi-scheduling-service-event
-Description: "Extension for service event id (palvelutapahtuman tunniste)."
+Description: "Extension for service event id (*palvelutapahtuman tunniste*)."
```
Comment (mrinnetmaki) [inline input/fsh/extensions/ServiceEvent.fsh:3]: 👍

**Review by mrinnetmaki — APPROVED** [review]
👏

**Pull request merged**